lit/docs/jobs.lit

```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
     \italic{Optional.} Keep logs for the last specified number of builds.
   }{job-build-log-retention-builds}
 
+  \define-attribute{minimum_succeeded_builds: number}{
+    \italic{Optional.} Keep at lease specified succeeded builds. For example,
+    if specifies \reference{job-build-log-retention-builds} to 5, and this
+    attribute to 1, say a job has a build history: 7(f), 6(f), 5(f), 4(f),
+    3(f), 2(f), 1(s), where f means failed and s means succeeded, then builds 2
+    and 3 will be reaped, because it retains 5 build logs, and at lease 1
+    succeeded build log. Default is 0.
+  }{job-build-log-retention-minimum-succeeded-builds}
+
```

Comment:
I wasn't involved at all with the original PR (which is probably a good thing now - fresh eyes and all  👀 ) so this paragraph reads a bit confusing to me! Going to suggest a multi-line rewrite, hopefully this works...